cinem_PD:
  kp: 2.1
  kd: 0.019

cinem_PI:
  kp: 1.1
  ki: 0.0415

cinem_Lyap:
  kp: 2.2

pid:
  x_kp: 2.95
  x_ki: 1.65
  x_kd: 0.0009
  y_kp: 2.95
  y_ki: 1.65
  y_kd: 0.0009
  theta_kp: 2.2
  theta_ki: 0.05
  theta_kd: 0.0000
